package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/SceneImporter.cs

```diff
@@ -338,9 +321,10 @@ public ImportException(string message, System.Exception innerException)
 
       if (ShouldYield(targetTime, timer)) { yield return null; ResetTimer(timer); }
 
-      Profiler.BeginSample("USD: AfterBuildPrimMap");
-      if (AfterBuildPrimMap != null) {
-        AfterBuildPrimMap(primMap, scene, usdPrimRoot, root, importOptions);
+      Profiler.BeginSample("USD: PostProcessHierarchy");
+      foreach( var processor in root.GetComponents<IImportPostProcessHierarchy>())
+      {
+        processor.PostProcessHierarchy(primMap, importOptions);
```

Comment:
We should wrap all callbacks in a try catch, like this:

```
try {
   processor.PostProcessHierarchy(primMap, importOptions);
} catch (Exception ex) {
   Debug.LogException(ex);
}
```

And I just noticed the old code wasn't doing this, but it really should have been.

Reply:
ok, wrapped.